docs/install/local/README.md

```diff
@@ -56,30 +56,49 @@ xcode-select --install
 1. Create a directory to be the home of your FlowForge install. For example: `/opt/flowforge`
```

Comment:
```suggestion
1. Create a `flowforge` directory to be the base of your FlowForge install. 
```

Reply:
I've updated this to match the installer/README.md with examples for both (which I want to keep because it adds context to the exampels) and changed `home` to `base`